data/api/v2/full_spec_deref.json

```diff
@@ -24360,7 +24360,7 @@
                     "type": "array"
                   },
                   "is_created": {
-                    "description": "Whether the deletion request is fully created or not.",
+                    "description": "Whether the deletion request is fully created or not. It can take some minutes to fully create a deletion request depending on the target query and timeframe.",
```

Comment:
```suggestion
                    "description": "Whether the deletion request is fully created or not. It can take several minutes to fully create a deletion request depending on the target query and timeframe.",
```